### PR TITLE
feat: image updates: add tf 2.7; security tf 2.4, 2.5, 2.6; fix PTL.

### PR DESCRIPTION
**Pull request by ioga**
## Description

Env image bump corresponding to https://github.com/determined-ai/environments/pull/125

## Test Plan

Should be covered with CI tests. Manual checks:
- "Everything" should still work.
- Run your favorite keras model on TF 2.7 image.
- Run PTL models using metrics from torchmetrics e.g. `e2e_tests/tests/fixtures/pytorch_lightning_amp/`.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by mackrorysd — APPROVED** [review]

**Comment by ioga** [comment]
Tests are passing, NGC doesn't have any severe vulnerabilities. admin merging this because upstream branch
